internal/apis/kaas/implementation/endpoints.go

```diff
@@ -10,4 +10,6 @@ var (
 
 	EndpointInstancePools = "/1/public_clouds/{public_cloud_id}/projects/{public_cloud_project_id}/kaas/{kaas_id}/instance_pools"
 	EndpointInstancePool  = "/1/public_clouds/{public_cloud_id}/projects/{public_cloud_project_id}/kaas/{kaas_id}/instance_pools/{kaas_instance_pool_id}"
+
+	EndpointApiserverParams = "/1/public_clouds/{public_cloud_id}/projects/{public_cloud_project_id}/kaas/{kaas_id}/apiserver/params"
```

Comment:
nit: maybe `/params` is not necessary in the endpoint and redundant with the field params